app/blueprints/round/routes.py

```diff
@@ -134,15 +135,24 @@ def edit_round(round_id):
     return render_template("round.html", **params, error=error)
 
 
-@round_bp.route("/<round_id>/clone")
-def clone_round(round_id, fund_id):
+@round_bp.route("/<round_id>/clone", methods=["POST"])
+def clone_round(round_id):
     """
     Clone an existing round.
     """
+    fund_id = request.form.get("fund_id")
```

Comment:
Instead of manually writing/managing/accessing form data, please can we create a new `CloneApplicationRoundForm` class in forms.py and use that.

Let's consistently use wtforms to handle all of our forms, because then we are consistently getting things like CSRF protection and other validation.

Reply:
Done updated